pkgs/ffigen/lib/src/code_generator/func.dart

```diff
@@ -176,12 +176,9 @@ $dartReturnType $enclosingFuncName($dartArgDeclString) => $funcImplCall;
       final isLeafString = isLeaf ? 'isLeaf:true' : '';
 
       // Write enclosing function.
-      s.write('''
-$dartReturnType $enclosingFuncName($dartArgDeclString) {
-  return $funcImplCall;
-}
-
-''');
+      s.write("$dartReturnType $enclosingFuncName($dartArgDeclString) {\n");
```

Comment:
This doesn't change anything?

Prefer `'''` strings instead of adding `\n`.

Reply:
Ah, I had originally changed something in the middle of this string so I split it for convenience, but I ended up reverting it. I'll revert this part too.